src/services/message.service.js

```diff
@@ -0,0 +1,15 @@
+const { Message } = require('../models/message.model');
+const { User } = require('../models/user.model');
+
+const getAllByRoom = async (roomId) => {
+  return Message.findAll({
+    where: { roomId },
+    include: [{ model: User, required: true }],
+  });
```

Comment:
Consider adding error handling for the `Message.findAll` operation to manage potential errors and prevent unhandled promise rejections.